now.json

```diff
@@ -10,7 +10,8 @@
     "env": {
       "ARAGON_FORTMATIC_API_KEY": "@aragon-client-fortmatic-testnet-api-key",
       "ARAGON_PORTIS_DAPP_ID": "@aragon-client-portis-dapp-id",
-      "ARAGON_SENTRY_DSN": "@aragon-client-sentry-dsn"
+      "ARAGON_SENTRY_DSN": "@aragon-client-sentry-dsn",
+      "ARAGON_ETH_SUBSCRIPTION_EVENT_DELAY": "3000"
```

Comment:
Also, should this be 5000 as in `now-mainnet.json`?

Reply:
Just guessing, but Rinkeby is used a lot less and hopefully has less sync problems.